src/Mod/Path/PathScripts/PostUtilsArguments.py

```diff
@@ -283,6 +313,12 @@ def init_shared_values(values):
     # The starting axis precision is 3 digits after the decimal point.
     #
     values["AXIS_PRECISION"] = 3
+    #
+    # How far to move up (in millimeters) in the Z axis when chipbreaking
+    # with a G73 command.
+    #
+    values["CHIPBREAKING_AMOUNT"] = Units.Quantity(0.25, FreeCAD.Units.Length)
```

Comment:
```suggestion
    values["CHIPBREAKING_AMOUNT"] = Units.Quantity(0.25, Units.Length)
```
Either one or another